metals/src/main/scala/scala/meta/internal/builds/NewProjectProvider.scala

```diff
@@ -0,0 +1,359 @@
+        case path if path.itemId == parentDir.id =>
+          askForPath(from.parent)
+        case path =>
+          askForPath(from.resolve(path.itemId))
+      }
+  }
+
+  // scala/hello-world.g8 -> hello-world
+  private def nameFromPath(g8Path: String) = {
+    g8Path.replaceAll(".*/", "").replace(".g8", "")
+  }
+}
+
+object NewProjectProvider {
+
+  val custom: MetalsQuickPickItem = MetalsQuickPickItem(
+    id = "custom",
+    label = "Custom",
+    description = "Enter template manually"
+  )
+
+  val more: MetalsQuickPickItem = MetalsQuickPickItem(
+    id = "more",
+    label = "Discover more...",
+    description = "From github.com/foundweekends/giter8/wiki/giter8-templates"
+  )
+
+  val back: MetalsQuickPickItem = MetalsQuickPickItem(
+    id = "back",
+    label = "Back",
+    description = "Back to curated Metals templates"
+  )
+
+  def curatedTemplates(icons: Icons): Seq[MetalsQuickPickItem] = {
+    Seq(
+      MetalsQuickPickItem(
+        id = "scala/hello-world.g8",
+        label = "scala/hello-world.g8",
+        description = "A template to demonstrate a minimal Scala application"
+      ),
+      MetalsQuickPickItem(
+        id = "scala/scalatest-example.g8",
+        label = "scala/scalatest-example.g8",
+        description = "A template for trying out ScalaTest"
+      ),
+      MetalsQuickPickItem(
+        id = "akka/akka-scala-seed.g8",
+        label = "akka/akka-scala-seed.g8",
+        description = "A minimal seed template for an Akka with Scala build"
+      ),
+      MetalsQuickPickItem(
+        id = "zio/zio-project-seed.g8",
+        label = "zio/zio-project-seed.g8",
+        description = "A template for ZIO"
+      ),
+      MetalsQuickPickItem(
+        id = "playframework/play-scala-seed.g8",
+        label = "playframework/play-scala-seed.g8",
+        description = "Play Scala Seed Template"
+      ),
+      MetalsQuickPickItem(
+        id = "lagom/lagom-scala.g8",
+        label = "lagom/lagom-scala.g8",
+        description = "A Lagom Scala seed template for sbt"
+      ),
+      MetalsQuickPickItem(
+        id = "scala-native/scala-native.g8",
+        label = "scala-native/scala-native.g8",
+        description = "Scala Native"
+      ),
+      MetalsQuickPickItem(
+        id = "lampepfl/dotty.g8",
+        label = "lampepfl/dotty.g8",
+        description = "A template for trying out Dotty"
+      ),
+      MetalsQuickPickItem(
+        id = "http4s/http4s.g8",
+        label = "http4s/http4s.g8",
+        description = "Simple http4s example"
+      ),
+      MetalsQuickPickItem(
+        id = "scalameta/mill-scala-seed.g8",
+        label = "scalameta/mill-scala-seed.g8",
+        description = "A Scala template for the Mill build tool"
+      ),
+      MetalsQuickPickItem(
+        id = "scalameta/gradle-scala-seed.g8",
+        label = "scalameta/gradle-scala-seed.g8",
+        description = "A Scala template for the Gradle build tool"
+      ),
+      MetalsQuickPickItem(
```

Comment:
I added a couple of new templates as well as one done by my colleague around Akka and kubernetes. The plan is to keep this list small so that we can be sure of the "curated" templates. It's always possible to manually enter a template or pick one from the list in https://github.com/foundweekends/giter8/wiki/giter8-templates

Let me know if any templates could benefit of being added to the list here.